backend/server.py

```
@@ -7874,6 +7874,123 @@ async def legacy_scheduler_toggle(
+
+class SamChatRequest(BaseModel):
+    text: str = Field(..., min_length=1, max_length=2000, description="Transcribed voice command")
+    session_id: str = Field(default="default", max_length=64)
+
+
+class SamSpeakRequest(BaseModel):
+    text: str = Field(..., min_length=1, max_length=2000, description="Text to synthesise as SAM's voice")
+
+
+@app.get("/agent/voice/status")
+async def voice_status_backend(user: dict = Depends(get_current_user)):
```

Ruff (0.15.18)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)